x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/results/AutodetectResultTests.java

```diff
@@ -48,15 +48,15 @@ protected AutodetectResult createTestInstance() {
         FlushAcknowledgement flushAcknowledgement;
         String jobId = "foo";
         if (randomBoolean()) {
-            bucket = new Bucket(jobId, new Date(randomNonNegativeLong()), randomNonNegativeLong());
+            bucket = new Bucket(jobId, new Date(randomLongBetween(0, 3000000000000L)), randomNonNegativeLong());
```

Comment:
Would it make sense to introduce a `#randomDate()` method here as well similarly to `AutodetectResultProcessorIT`?

Reply:
yes